app/views/users/piv_cac_recommended/show.html.erb

```diff
@@ -0,0 +1,33 @@
+<% self.title = t('titles.piv_cac_setup.upsell') %>
+
+<%= render AlertIconComponent.new(icon_name: :info_question) %>
+
+<%= render PageHeadingComponent.new.with_content(t('titles.piv_cac_setup.upsell')) %>
+
+<p>
+  <%= @recommended_presenter.info %>
+</p>
+
+<%= render ButtonComponent.new(
+      action: ->(**tag_options, &block) do
+        button_to(
+          login_piv_cac_recommended_add_path,
+          **tag_options,
+          &block
+        )
+      end,
+      big: true,
+      class: 'margin-bottom-3',
+    ).with_content(t('two_factor_authentication.piv_cac_upsell.add_piv')) %>
+
+<%= render ButtonComponent.new(
+      action: ->(**tag_options, &block) do
+        button_to(
+          login_piv_cac_recommended_skip_path,
+          **tag_options,
+          &block
+        )
+      end,
+      class: 'usa-button usa-button--unstyled',
```

Comment:
`usa-button` is applied automatically by `ButtonComponent`. `usa-button--unstyled` could be applied with the abstracted constructor property

```suggestion
      unstyled: true,
```